classification.yaml

```diff
@@ -0,0 +1,791 @@
+DMS-MWST-REQ-0031:
+  status: unclear
+  notes:
+    I _think_ we might satisfy this via skipping quanta whose datasets exist in the current output RUN collection (using the same RUN collection is what guarantees config/software/input consistency).
+    But that's a very different description of the functionality from what's in the requirement.
+    Might want to check with GPDF to see if he buys this interpretation.
+    We might also be able to have more sophisticated skip-existing logic when provenance stuff has landed, which would more clearly satisfy the use case of the requirement even if it's not really more similar in terms of description.
+DMS-MWST-REQ-0004:
+  status: feature-complete
+  notes:
+    Pipeline YAML files satisfy this requirement (they don't even need to be ordered; the system can order them itself).
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0005:
+  status: feature-complete
+  notes:
+    Pipeline YAML files satisfy this requirement.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0006:
+  status: feature-complete
+  notes:
+    Grouping information is actually embedded in the PipelineTask definition, not directly the Pipeline, but since a Pipeline include PipelineTasks, this is still satisfied.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0007:
+  status: feature-complete
+  notes:
+    See notes for DMS-MWST-REQ-0006.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0008:
+  status: feature-complete
+  notes:
+    Satisfied by PipelineTask being a subclass of Task.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0003:
+  status: feature-complete
+  notes:
+    The Pipeline Python API is not heavily utilized, but it certainly does exist and was conceived of as a public interface.
+DMS-MWST-REQ-0010:
+  status: feature-complete
+  notes:
+    This is just how pipetask and BPS work.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0011:
+  status: feature-complete
+  notes:
+    This is just how pipetask and BPS work.
+  jira:
+  - LVV-T1982
+  - LVV-T1983f
+DMS-MWST-REQ-0012:
+  status: feature-complete
+  notes:
+    QuantumGraph generation guarantees this.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0013:
+  status: feature-complete
+  notes:
+    This is how pipetask and BPS work.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0009:
+  status: unclear
+  notes:
+    It looks like someone wants to delete this as redundant with -0013.  But we can do it all.
+DMS-MWST-REQ-0014:
+  status: feature-complete
+  notes:
+    Spec for this is almost impossible to understand on its own, but description makes it clear that we've satisfied this with the PipelineTask design.
+  jira:
+  - LVV-T1982
+  - LVV-T1983
+DMS-MWST-REQ-0015:
+  status: unclear
+  notes:
+    I was either not aware of this one or had forgotten about it.  I am skeptical that Pipeline is the right place for this functionality, but the lack of a motivating use case makes it hard to reason about.
```

Comment:
I am hoping someone else can remember why we added this.

Reply:
No idea. The "Supertask" requirements don't have annotated use cases.

Reply:
IIRC the idea was to be able to insert steps like the faro metric generation ones (with different concerns and mutability properties) into the main algorithmic pipeline definition. I'd think a configuration-based insertion (which I believe is what we have now) is sufficient for this rather than having an API-based insertion.